### PR TITLE
Tag Bio v0.4.0 [https://github.com/BioJulia/Bio.jl]

### PR DESCRIPTION
**Pull request by bicycle1885**

### CODE REVIEW

**Comment by tkelman** [comment]
`test/tools/runtests.jl` still does `using Compat` but Compat was removed from `REQUIRE`

**Comment by bicycle1885** [comment]
Okay, tagged a new version v0.4.1.

File: Bio/versions/0.4.1/sha1 [hunk 1]
```diff
@@ -0,0 +1 @@
+ecba7c7681861d6a078b835c6fb350053e1b1e97
```
Comment (tkelman) [inline Bio/versions/0.4.1/sha1]: looks like you've only pushed the git tag for this to your fork so far

Comment (tkelman) [inline Bio/versions/0.4.1/sha1]: and I don't see https://github.com/BioJulia/Bio.jl/commit/ecba7c7681861d6a078b835c6fb350053e1b1e97 on any branch?

Comment (bicycle1885) [inline Bio/versions/0.4.1/sha1]: Sorry I made a mistake but now fixed.

Comment (tkelman) [inline Bio/versions/0.4.1/sha1]: looks like you tagged a commit that was inside a PR - probably better to be sure you're on the release branch before tagging

Comment (bicycle1885) [inline Bio/versions/0.4.1/sha1]: Definitely.

**Pull request merged**